hack/boilerplate/boilerplate.Dockerfile.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright YEAR The Machine Controller Authors.
```

Comment:
I think this must be year because the string `YEAR` is deemed a placeholder by the verify script, the string `2019` not. If we merge it like this, the presubmit will stop working in 2020